src/main/scala/mesosphere/marathon/api/akkahttp/EntityMarshallers.scala

```diff
@@ -150,12 +153,12 @@ object EntityMarshallers {
   implicit val messageMarshaller = playJsonMarshaller[Rejections.Message]
   implicit val appInfoMarshaller = playJsonMarshaller[AppInfo]
   implicit val infoMarshaller = playJsonMarshaller[MarathonInfo]
-  implicit val infoUnmarshaller = playJsonUnmarshaller[MarathonInfo]
   implicit val metricsMarshaller = internalToRamlJsonMarshaller[TickMetricSnapshot, Metrics]
   implicit val loggerChangeMarshaller = playJsonMarshaller[LoggerChange]
   implicit val loggerChangeUnmarshaller = playJsonUnmarshaller[LoggerChange]
   implicit val stringMapMarshaller = playJsonMarshaller[Map[String, String]]
   implicit val pluginDefinitionsMarshaller = playJsonMarshaller[PluginDefinitions]
+  implicit val queueMarashaller = internalToRamlJsonMarshaller[(Seq[QueuedInstanceInfoWithStatistics], Boolean, Clock), raml.Queue]
```

Comment:
Have you considered to use a case class instead of tuple3 here?

Reply:
I did and I couldn't come up with any good name. So I left it a tuple

Reply:
Is `Clock` the correct type? Why is this not a simple `playJsonMarshaller[raml.QueueResponse]`?

Reply:
because `Seq[QueuedInstanceInfoWithStatistics]` is filtered with  `Boolean, Clock` to form the `Queue.raml` response: https://github.com/mesosphere/marathon/blob/master/src/main/scala/mesosphere/marathon/raml/QueueInfoConversion.scala#L21 
Didn't want to change current implementation - we can do it later.

Reply:
Oh man. You are right.